Lib/test/test_hashlib.py

```diff
@@ -50,7 +50,7 @@ def hexstr(s):
     return r
 
 
-URL = "http://www.pythontest.net/hashlib/{}.txt"
+URL = f"{TEST_HTTP_URL}/hashlib/{{}}.txt"
```

Comment:
Don't do that. It will break the test if TEST_HTTP_URL is changed to example.com or anything else.